src/routes/comments.ts

```diff
@@ -1,20 +1,86 @@
-import express from "express";
-import { getComments } from "../database/comment";
+import express, { Request, Response } from "express";
+import mongoose from "mongoose";
+import * as commentService from "../database/comment";
 
 const router = express.Router();
 
-router.get("/", async (req, res) => {
+router.post("/", async (req: Request, res: Response) => {
   try {
-    console.log("In Item...");
-    const comments = await getComments();
+    const comment = await commentService.createComment(req.body);
+    res.status(201).send(comment);
+  } catch (error: unknown) {
+    res.status(500).send({ error: "Internal Error" });
+  }
+});
+
+router.get("/", async (_req: Request, res: Response) => {
+  try {
+    const comments = await commentService.getAllComments();
+    if (comments.length > 0) {
+      return res.status(200).send(comments);
+    } else {
+      return res.status(404).send();
```

Comment:
An empty comment list shouldn't return a 404 status code. Leave it for the client to handle.